Python/Product/EnvironmentsList/ToolWindow.xaml.cs

```diff
@@ -556,6 +557,25 @@ await StartRefreshDBAsync(view)
             }
         }
 
+        private static void SelectListBoxItem(DependencyObject child, bool isSelected) {
+            while (child != null) {
+                var lbi = child as ListBoxItem;
+                if (lbi != null) {
+                    lbi.IsSelected = true;
```

Comment:
Will fix this - should be `isSelected` (or perhaps removed completely... since it works despite this...)